libjiffy/CMakeLists.txt

```diff
@@ -232,17 +232,27 @@ if (BUILD_STORAGE OR BUILD_DIRECTORY)
           src/jiffy/storage/types/binary.cpp
           src/jiffy/storage/types/byte_string.h
           src/jiffy/storage/types/byte_string.cc)
-
-  add_dependencies(jiffy thrift_ep memkind_ep)
-  message(STATUS "numalibrary = ${NUMA_LIBRARY}")
-  target_link_libraries(jiffy ${THRIFT_LIBRARY}
-          ${THRIFTNB_LIBRARY}
-          ${LIBEVENT_LIBRARY}
-          ${AWSSDK_LINK_LIBRARIES}
-          ${MEMKIND_LIBRARY}
-          ${NUMA_LIBRARY}
-          ${CMAKE_DL_LIBS}
-          ${CMAKE_THREAD_LIBS_INIT})
+  if (BUILD_MEMKIND_SUPPORT)
```

Comment:
You can replace this with HEAP_MANAGER_EP and HEAP_MANAGER_LIBRARY.

Reply:
We should keep it here because libnuma will be linked when we use memkind.

Reply:
You can add NUMA_LIBRARY to HEAP_MANAGER_LIBRARY..